docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/first-time-configuration-for-admin-api-1x.md

```diff
@@ -0,0 +1,96 @@
+# First-Time Configuration for Admin API 1.x
+
+After [Installing the Admin
+API](../admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md),
+there are a couple manual steps that must be completed before the application
+can be used.
+
+* [1\. Launch the Application](#1-launch-the-application)
+* [2\. Create the First API Client](#2-create-the-first-api-client)
+* [Optional - Self-Signed Certificates](#optional-self-signed-certificates)
+
+# 1\. Launch the Application
+
+The database is now initialized and ready to function. Visit the configured URL
+from a browser (or launch from IIS or Docker Desktop if using those) and verify
+the application is running. The root page (`/`) should return JSON with the API
+and Build versions. Additionally you can visit `/swagger/index.html` if Swagger
+is enabled in appsettings and view the web-based documentation.
+
+**Sample Response from /**
+
+```
+{
+  "version": "1.0",
+  "build": "1.0.0.0"
+}
+```
+
+![](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v1/image2022-8-3_17-32-20.png)
+
+# 2\. Create the First API Client
+
+In order to authenticate with the API you must first register a client key and
+secret. This is a client for the _Admin API_, not an Application and key /
+secret for interfacing with the Ed-Fi ODS / API.
+
+Client registration is done by sending a url-encoded form request to
+`/connect/register.` The ability to register new clients is similar to adding
+users in Admin App. By default, **this endpoint is only available when no
+clients have been created**. After the first client is created, the endpoint is
+disabled. The endpoint can be re-enabled with the
+`Authentication/AllowRegistration`  setting or environment variable (`false`  by
+default).
+
+> [!CAUTION] Since the `/connect/register`  endpoint does not require any
+> authentication, it is recommended that it remains disabled when not
+> registering a new client. Admin API v1.0 does not include any scope
+> limitations, so all clients are able to utilize all endpoints.
+
+The sample below is using `curl,`  however this request can be performed from
+Postman, the Swagger site, or from your application.
+
+```
+curl -X POST https://your-admin-api/connect/register -H "Content-Type: application/x-www-form-urlencoded" -d "ClientId=YourClientId&ClientSecret=YourClientSecret&DisplayName=YourDisplayName"
+```
```

Comment:
These long lines in code blocks will often force the user to scroll horizontally. I have been converting many of them to have line breaks using the PowerShell line continuation character, which is a single backtick.

```powershell
curl -X POST https://your-admin-api/connect/register `
    -H "Content-Type: application/x-www-form-urlencoded" `
    -d "ClientId=YourClientId&ClientSecret=YourClientSecret&DisplayName=YourDisplayName"
```